doc/sphinx/make-your-own-compositor.md

```diff
@@ -0,0 +1,248 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Using Mir Libraries to Build Your Own Wayland Compositor
```

Comment:
```suggestion
# Tutorial
```

Reply:
Even if we don't call this "Tutorial" it should be called "Build a Wayland Compositor". I think Mir is implied 😉

Reply:
I agree this is verbose, but "Build" suggest checkout && make, we want something more along the lines of "Writing"...

"Creating A First Compositor"
"Writing A Simple Compositor"

Reply:
How does "Writing your first Wayland Compositor" sound?